de.dlr.sc.virsat.model.extension.requirements/concept/concept_v0_7.xmi

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="ASCII"?>
-<dvlm_c:Concept xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:dvlm_c="http://www.virsat.sc.dlr.de/dvlm/v10/c" xmlns:dvlm_cp="http://www.virsat.sc.dlr.de/dvlm/v10/cp" xmlns:dvlm_cppd="http://www.virsat.sc.dlr.de/dvlm/v10/cp/cppd" name="de.dlr.sc.virsat.model.extension.requirements" description="Extension for Requirement Specification" version="0.7" displayName="Requirements" beta="true">
```

Comment:
I think some of your test case failiures might come from unintentionally modified fieles. Here, you commited a concept file that was regenerated with an old version of our IDE... You can see this because the DVLM version decreased from version 10 to version 9.... This is why you requirements migrator test fails...
So if you commit your changes, please make sure that you only add changes on those files, that you actually modified on purpose. For this pull request you can go through the modified files and reset (replace with branch -> development) the files you didn't want to change... :)